cloudinit/net/dhcp.py

```diff
@@ -293,12 +316,17 @@ def parse_static_routes(rfc3442):
         ("169.254.169.254/32", "130.56.248.255"), ("0.0.0.0/0", "130.56.240.1")
     ]
 
+    sr2 = parse_state_routes("24.191.168.128 192.168.128.1,0 192.168.128.1")
+    sr2 = [
+        ("191.168.128.0/24", "192.168.128.1"), ("0.0.0.0/0", "192.168.128.1")
+    ]
+
     Python version of isc-dhclient's hooks:
        /etc/dhcp/dhclient-exit-hooks.d/rfc3442-classless-routes
     """
     # raw strings from dhcp lease may end in semi-colon
     rfc3442 = rfc3442.rstrip(";")
-    tokens = rfc3442.split(',')
+    tokens = re.split(r",| |\.", rfc3442)
```

Comment:
I think this regexp is hard to read and can be improved to `"[, .]"`, what do you think?

Reply:
ha, you are right. Give me a moment.